config/karma.js

```diff
@@ -0,0 +1,52 @@
+const karma = require('karma');
+const getPortSync = require('get-port-sync');
+
+const port = getPortSync();
+
+module.exports = function(config) {
+	config.set({
+		autoWatch: false,	// default: true
+		basePath: '../',
+		browserDisconnectTimeout: 45000,	// default: 2000
```

Comment:
@jvercide-tc these could likely be reduced significantly as this isn't a large application

Reply:
@cjardine-tc updated